etc/cvmfs/domain.d/sdcc.bnl.gov.conf

```diff
@@ -1,10 +1,15 @@
-# Replace this file with a symlink to ../osghosted.conf once the selinux
-#  issue in https://sft.its.cern.ch/jira/browse/CVM-1922 is no longer a
-#  concern.
-
-# refer to all variables set by osghosted.conf so cvmfs will read them
+# These are here so cvmfs will notice them if common.conf sets them
 CVMFS_USE_CDN="$CVMFS_USE_CDN"
 CVMFS_HTTP_PROXY="$CVMFS_HTTP_PROXY"
 CVMFS_FALLBACK_PROXY="$CVMFS_FALLBACK_PROXY"
-CVMFS_SERVER_URL="$CVMFS_FALLBACK_PROXY"
-. ../osghosted.conf
+. ../common.conf
+
+if [ "$CVMFS_USE_CDN" = "yes" ]; then
+    CVMFS_SERVER_URL="http://s1bnl-cvmfs.openhtc.io/cvmfs/@fqrn@;http://s1fnal-cvmfs.openhtc.io/cvmfs/@fqrn@;http://s1osggoc-cvmfs.openhtc.io/cvmfs/@fqrn@"
+else
+    CVMFS_SERVER_URL="http://cvmfs-s1bnl.opensciencegrid.org:8000/cvmfs/@fqrn@;http://cvmfs-s1fnal.opensciencegrid.org:8000/cvmfs/@fqrn@;http://cvmfs-s1goc.opensciencegrid.org:8000/cvmfs/@fqrn@"
+fi
+CVMFS_PUBLIC_KEY="$CVMFS_MOUNT_DIR/$CVMFS_CONFIG_REPOSITORY/etc/cvmfs/keys/sdcc.bnl.gov.pub"
```

Comment:
Should this key be put in a domain directory?

Reply:
Likewise to above comment, except it wasn't in the egi branch that way but it was in the master branch (because I put it there recently to work with the new auto key download feature in [manage-replicas](https://github.com/cvmfs-contrib/cvmfs-manage-replicas/blob/master/manage-replicas.conf#L24).  That feature reads either individual keys matching a domain name or a directory matching a domain name.